src/ray/core_worker/core_worker.cc

```diff
@@ -3460,10 +3458,10 @@ void CoreWorker::HandleReportGeneratorItemReturns(
       request,
       /*execution_signal_callback*/
       [reply,
-       worker_id = std::move(worker_id),
-       generator_id = std::move(generator_id),
+       worker_id,
```

Comment:
I'm not sure if it's good, my concern is, even if worker ids are POD type, we could have other new data members like std::string.

Reply:
I think they're trivially copyable like small array of 8-12 or so bytes, no strings or anything

Reply:
Yeah my point is: it's good for now since it's POD, but if we add non-trivially copiable fields in the future, we have to do move everywhere; that's the reason why I prefer move if possible.

Reply:
updated to move